index.html

```diff
@@ -69,6 +78,242 @@ <h1 class="primary-txt">
                 </ul>
             </div>
         </section>
+        <section class="works">
+            <div class="card-works">
+                <div class="snapshoot-portfolio bg-1">
+                </div>
+                <div class="left-block">
+                    <div>
+                        <h1 class="primary-txt">
+                            Tonic
+                        </h1>
```

Comment:
- [ ] For proper semantics and improved SEO, it is advised to use one single h1 per web page to emphasize only one most important heading. kindly change the work card title to a `<h2....h6>` tag to also reduce spamming. (having multiple h1) in a webpage